lib/Foo/BlessArray.pm

```diff
@@ -1,21 +1,40 @@
 use v5.40;
 
 package Foo::BlessArray {
+    use Const::Fast qw(const);
+
+    const my $foo => 0;
```

Comment:
Using constants is a great idea!
I wanted to use the constant pragma, expecting constant folding. Since constant is a core module, I also liked that it didn't require any additional installation.

```perl
constant {
   FOO => 0,
   BAR => 1,
   BAZ => 2,
};
```